Gruntfile.js

```diff
@@ -35,6 +32,11 @@ module.exports = function (grunt) {
 
     'dev:prehook': ['process:api:start'],
     'dev:posthook': ['process:api:stop'],
+
+    'test:crossbrowser' : ['build', 'assemble', 'connect', 'sauce_tunnel', 'mochaTest', 'sauce_tunnel_stop']
   });
 
+  // loads library specific tasks
```

Comment:
Can we add an additional comment here for why this was done? ... Execution order matters. We want local task configuration to overwrite easy-build task configuration.